### PR TITLE
Getting Started with Synthetic Monitoring > Multistep API Tests

### PR DESCRIPTION
**Pull request by alai97**
Adding a section on multistep API tests.

### What does this PR do?

Revamp the Getting Started with API Tests page with updated content about multistep API tests.

### Motivation

DOCS-1979

### Preview

https://docs-staging.datadoghq.com/alai97/synthetics-api-tests-getting-started/getting_started/synthetics/api_test

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by kayayarai — CHANGES_REQUESTED** [review]
Looks good! Just two things

File: content/en/getting_started/synthetics/api_test.md [hunk 1]
```diff
@@ -93,15 +94,43 @@ You can set time for your alert notification to re-notify if the alert has not b
 
 When you're ready to run your test, click **Save Test**. 
 
-### Test results
+## Create a multistep API test
 
-The API test details page displays an overview of the test configuration, the global uptime associated with the tested endpoint by location, graphs about response time and network timings, and a list of test results and events.
+Multistep API tests also **monitor your API endpoints** and **alert you** when they become too slow or fail. These tests verify that your services are responding to HTTP requests and meeting any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/api_test.md:99]:
```suggestion
Multistep API tests **monitor your API endpoints** and **alert you** when they become too slow or fail. These tests verify that your services are responding to HTTP requests and meeting any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**. 
```

File: content/en/getting_started/synthetics/api_test.md [hunk 2]
```diff
@@ -93,15 +94,43 @@ You can set time for your alert notification to re-notify if the alert has not b
...
+
+1. In the Datadog application, hover over **[UX Monitoring][2]** in the left hand menu and select **Synthetic Test**.
+2. In the top right corner, click the **New Test** button.
+3. Select **Multistep API test**.
+4. Define your multistep API request like you would with an API test.
+5. In **Define steps**, click **Create Your First Step**. 
+
+    - Add a name to your test HTTP request step.
+    - Specify the URL you want to query. 
+    - You can select **Advanced Options** to add request options, request body, certificates, and proxies.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/api_test.md:113]:
```suggestion
    -  Select **Advanced Options** to add request options, request body, certificates, and proxies.
```

File: content/en/getting_started/synthetics/api_test.md [hunk 3]
```diff
@@ -93,15 +94,43 @@ You can set time for your alert notification to re-notify if the alert has not b
...
+    - Click **Test URL**. 
+    - You can optionally add new assertions, enable execution parameters, and extract variables from the response content.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/api_test.md:115]:
```suggestion
    - Optionally, add new assertions, enable execution parameters, and extract variables from the response content.
```

File: content/en/getting_started/synthetics/api_test.md [hunk 4]
```diff
@@ -93,15 +94,43 @@ You can set time for your alert notification to re-notify if the alert has not b
...
+    - Click **Save Step**.
+
+6. Click **Add Another Step** to add another step. You can create up to ten steps. 
+7. Configure the rest of your test conditions including test frequency, alerting conditions, and alert message.
+
+When you're ready to run your test, click **Save Test**.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/api_test.md:121]:
```suggestion
When you're ready to create your test, click **Save Test**. 
```

File: content/en/getting_started/synthetics/api_test.md [hunk 5]
```diff
@@ -93,15 +94,43 @@ You can set time for your alert notification to re-notify if the alert has not b
...
+## Test results
+
+The API test and multistep API test detail pages display an overview of the test configuration, the global uptime associated with the tested endpoint by location, graphs about response time and network timings, and a list of test results and events.
 
 To troubleshoot a failed test, scroll down to **Test Results** and click on a failing test result. Review failed assertions and response details such as status code, response time, and associated headers and body to diagnose the issue.
 
 {{< img src="getting_started/synthetics/api-test-failure.png" alt="API test failure"  style="width:90%;">}}
 
-With Datadog's [APM integration with Synthetic Monitoring][5], you can access the root cause of a failed test run with a trace generated from the test run.
+With Datadog's [APM integration with Synthetic Monitoring][5], you can access the root cause of a failed test run by looking at the trace generated from the test run in the **Traces** tab.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/api_test.md:133]:
```suggestion
With Datadog's [APM integration with Synthetic Monitoring][5], access the root cause of a failed test run by looking at the trace generated from the test run in the **Traces** tab.
```

**Review by sarina-dd — COMMENTED** [review]
Small suggestions 😸

File: content/en/getting_started/synthetics/api_test.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/getting_started/synthetics/api_test.md:99]:
```suggestion
Multistep API tests also **monitor your API endpoints** and **alert you** when they become too slow or fail. These tests verify that your services are executing HTTP requests and meeting any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**. 
```

[the remark this answers is not in the document]
Comment (sarina-dd) [reply content/en/getting_started/synthetics/api_test.md:99]: +1

**Comment by sarina-dd** [comment]
@alai97 A quick note on this page: it looks like quite a few of the screenshots are outdated. The one in configure a request section should be updated to reflect multiple request types now (SSL, TCP, DNS, etc.) The assertions image too as the right panel has changed (it no longer shows the request preview / has a few options for response, docs, and variables that are nifty!)

File: content/en/getting_started/synthetics/api_test.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/getting_started/synthetics/api_test.md:99]: How about this? @kayayarai @sarina-dd

File: content/en/getting_started/synthetics/api_test.md [hunk 6]
```diff
@@ -93,15 +94,43 @@ You can set time for your alert notification to re-notify if the alert has not b
...
 
-### Test results
+## Create a multistep API test
 
-The API test details page displays an overview of the test configuration, the global uptime associated with the tested endpoint by location, graphs about response time and network timings, and a list of test results and events.
+Multistep API tests also **monitor your API endpoints** and **alert you** when they become too slow or fail. These tests verify that your services are responding to HTTP requests and meeting any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**. 
+
+### Configure the test
+
+Creating a multistep API test slightly differs from creating an API test. You can link HTTP requests and create variables from the request response data in **Define steps**.
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/getting_started/synthetics/api_test.md:103]: Is "aPI" a typo here @kayayarai?

[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/getting_started/synthetics/api_test.md:103]: Yes _shakes fist at wireless keyboard_

File: content/en/getting_started/synthetics/api_test.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/getting_started/synthetics/api_test.md:99]:
```suggestion
Multistep API tests chain multiple HTTP requests together, allowing you to configure API tests on complex backend systems. 
```
I think we could whittle it down to this ^

[the remark this answers is not in the document]
Comment (alai97) [reply content/en/getting_started/synthetics/api_test.md:99]: yas

**Review by kayayarai — APPROVED** [review]

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**